hs_dataverse/management/commands/dataverse.py

```diff
@@ -0,0 +1,318 @@
+# -*- coding: utf-8 -*-
+
+"""
+Generate metadata and bag for a resource from Django
+
+"""
+import os
+import json
+import requests
+from django.core.management.base import BaseCommand
+from hs_core.models import BaseResource
+from hs_core.hydroshare.hs_bagit import create_bag_files
+from hs_core.tasks import create_bag_by_irods
+from hs_core.hydroshare import get_party_data_from_user
+from django_irods import icommands
+from hs_dataverse.utils import upload_dataset
+import tempfile
+import shutil
+
+BUFSIZE = 4096
+# helper functions to get different kinds of metadata
+
+
+# get owners. An owner is a user. see https://docs.djangoproject.com/en/3.0/ref/contrib/auth/
+def get_owner_data(resource):
```

Comment:
This function should be in your hs_dataverse directory, not in the management command.